ansible_builder/_target_scripts/introspect.py

```diff
@@ -5,7 +5,7 @@
 import yaml
 
 import requirements
-from pkg_resources import safe_name
+import importlib.metadata
```

Comment:
Can you please create a backward-compatible shim just like https://github.com/ansible/ansible-runner/pull/1224/files#diff-fe262d49ef98e3d8bda61b35f5546ef3f425fc52cfcb301be8e204a0b08d5707R1-R6?

Reply:
Can't do it that way (without a lot of pain anyway)- `introspect` gets copied into the target container and run standalone during the build, so it won't be able to access any other builder guts. It'll just need to do the import dance inline.

Reply:
Thinking about it a little more, so long as the EE Python is >= 3.9, what's already here should work fine. If we want to be able to build with <3.9, we should probably just inline the PEP503 + `_` normalization ala https://github.com/python/cpython/blob/3.10/Lib/importlib/metadata/__init__.py#L866-L871